transformer_engine/jax/cpp_extensions.py

```diff
@@ -1679,7 +1681,7 @@ def lowering(ctx, grad_outputs, softmax_outputs, *, scale_factor):
                                                          grad_outputs, softmax_outputs,
                                                          scale_factor)
 
-        return out # out is iterable already
+        return out    # out is iterable already
```

Comment:
nitpik: Normally, it is 2 whitespace before the `#` symbol.

Reply:
We had a yapf style config that specifies to use 4 whitespaces, but it is not added in the qa script. We should add them into the qa script to have the align style check. cc @jeng1220